CHANGELOG.rst

```diff
@@ -28,6 +31,8 @@ Internal Changes
 - **Notebooks**: Use the correct user labels when caching sessions.
 - **Helm chart**: Set the notebooks replica count to zero because it is replaced by the data services.
 - **Helm chart**: Add RBAC rules for Shipwright and Tekton resources in the Kubernetes cache.
+- **Amalthea**: Support using private images.
```

Comment:
but only from the renku-connected gitlab, right?

Reply:
afaik this just allows amalthea sessions to have an image pull secret defined. The wiring to actually set this is not in place in data svc or in the UI.
@olevski :point_up:

Reply:
Yes @Panaetius is correct Ralf. That is why this is under Amalthea and in the internal changes section.